.travis.yml

```diff
@@ -26,22 +26,21 @@ before_install:
         wget https://s3.amazonaws.com/octave-snapshot/public/octave-ubuntu-trusty-snapshot.tar.xz;
         sudo tar --extract --directory=/usr/local --strip-components=1 --file=octave-ubuntu-trusty-snapshot.tar.xz;
     fi
-  - sudo apt-get install -y python$PYTHON_VER-pip
+  - sudo apt-get install -y python$PYTHON_VERSION-pip
   - if [ "x$PYTAVE" = "xyes" ]; then
-        sudo apt-get install -y libboost-python-dev python$PYTHON_VER-numpy;
+        sudo apt-get install -y python$PYTHON_VERSION-dev python$PYTHON_VERSION-numpy;
```

Comment:
maybe we should leave a comment in the yml file then to remember to remove it later